charts/fleet-agent/fleet-agent/0.3.5/Chart.yaml

```diff
@@ -0,0 +1,12 @@
+annotations:
```

Comment:
@jiaqiluo we should completely delete charts/, assets/ and index.yaml and recreate with 2 `make validate` calls; that way, we can also cleanup artifacts devs have currently left behind in this release.